### DISCUSSION TITLE
Can I Record Slow-Mo at Exactly 240 FPS?

### DISCUSSION THREAD

**Discussion by AlexLandherr** [gopro/labs#556] [Q&A]
Is it at all possible to record slow-mo video on the HERO11 Black at exactly 240 fps (like with 24 fps = 24 Hz)? So it's 240 fps = 240 Hz?

Or is this not technically feasible at all? I just need to know.

**Comment by dnewman-gpsw** [comment]
Making 24.0Hz was not a small effort, and it has very clear customer benefits. Other integer frame are less critical, and 240fps is likely to be the least.  What is the technical benefit you gain from 240.0 fps vs 239.76fps?

**Reply by AlexLandherr** [reply]
I just wondered whether it was possible; if it's too laborious I won't take it up again.
